### DISCUSSION TITLE
Entity Relationship Diagram for OpenDCS Databases?

### DISCUSSION THREAD

**Discussion by agilmore2** [opendcs/opendcs#301] [Q&A]
Do ERDs exist for any of the DECODES database, computations, or the OpenTSDB version of database storage?

**Comment by MikeNeilson** [comment]
Not at present - that I'm aware of. It's one of the things I've poked at but haven't really setup anything. Was hoping to automated but probably good to just get something out there. Not like it changes that often.

**Comment by agilmore2** [comment]
Thanks.
